plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2925,22 +2925,22 @@ public void testTruncateTextTransform()
                 "('abcd', 1)," +
                 "('abxy', 2)," +
                 "('ab598', 3)," +
-                "('mommy', 4)," +
-                "('moscow', 5)," +
+                "('Kielce', 4)," +
+                "('Kiev', 5)," +
```

Comment:
This is much nicer :) But just wanted to point out that the preferred romanization is "Kyiv". Sadly, there's no timezone "Europe/Kyiv".